### PR TITLE
Security architecture becomes security concepts

### PR DESCRIPTION
**Pull request by chayim**
This is a placeholder PR, only to be merged if/when the PR linked below is merged.

https://github.com/sourcegraph/interviews/pull/146/files

### CODE REVIEW

File: handbook/engineering/hiring/software-engineer-security.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
    - Read through [our handbook](https://github.com/sourcegraph/about) to learn more about how we operate and to find answers to common questions that you might have.
 1. You write a short security assessment, taking less than **1hr** of your time, explaining a security vulnerability [sample problem](https://docs.google.com/document/d/1oXhjU_3y2uhpmWJ2hD0NTaqSanUAOlKDFFmjazAU3Rg/).
 1. We schedule remote interviews over video chat across multiple days.
-   - 1h **Security architecture:** We give you an open problem statement and you walk us through how you would solve the problem.
+   - 1h **Security concepts:** We give you an open problem statement and you walk us through how you would solve the problem.
```
Comment (nicksnyder) [inline handbook/engineering/hiring/software-engineer-security.md:30]: Is this description still accurate?

Comment (chayim) [reply handbook/engineering/hiring/software-engineer-security.md]: It is now. I added specifics on the second part of the discussion, as I feel we'd have a better candidate conversation if were this more open.

File: handbook/engineering/hiring/software-engineer-security.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ We are looking for an engineer who will proactively improve the security of our
 - You are [high agency](https://twitter.com/shreyas/status/1276956836856393728)
 - You communicate effectively in writing and documentation
 - You are able to directly contribute to our codebase and infrastructure (Go, TypeScript, Kubernetes, GCP)
+- You have at least 3 hours of overlap with [IDT](http://time.is/IDT) per week
```
Comment (nicksnyder) [inline handbook/engineering/hiring/software-engineer-security.md:14]: This is not clear enough. Which timeframe in IDT is the candidate supposed to overlap with?

Comment (chayim) [reply handbook/engineering/hiring/software-engineer-security.md]: Updated PR to indicate 0800-1800 IDT.

**Review by withdavidli — APPROVED** [review]

**Pull request closed without merging**